zinit-install.zsh

```diff
@@ -393,7 +393,7 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || {
                         command rm -f "${REPLY:t}.sig"
                     fi
 
-                    command mkdir -p ._zinit
+                    command mkdir -p ._zinit && echo '*' > ._zinit/.gitignore
```

Comment:
```suggestion
                    command mkdir -p ._zinit && echo '._zinit' > ._zinit/.gitignore
```

Ignoring *everything* will essentially not allow you to work off of the zinit plugin dir.
Ignoring `._zinit/` should be enough.

Reply:
Can this work? If `._zinit/.gitignore` is `*`, it will ignore `._zinit/*`. If `._zinit/.gitignore` is `._zinit`, it will ignore `._zinit/._zinit`. I think the first is our need?

Reply:
Oh yeah my bad. I read that wrong. All good.